app/ModelSerializers/Summit/Registration/SponsorBadgeScanCSVSerializer.php

```diff
@@ -52,17 +62,36 @@ public function serialize($expand = null, array $fields = array(), array $relati
         $values = parent::serialize($expand, $fields, $relations, $params);
         $sponsor = $scan->getSponsor();
         $sponsor_questions = $sponsor->getExtraQuestions();
+        $setting = $sponsor->getSummit()->getLeadReportSettingFor($sponsor);
+        $setting_columns = $setting->getColumns();
+
+        // remove not allowed string columns
+        foreach($values as $id => $value) {
+            if (!in_array($id, $setting_columns)) {
+                unset($values[$id]);
+            }
+        }
 
         if (isset($params['ticket_questions'])) {
```

Comment:
serialization logic should consider following settings columns values
extra_questions: [] and attendee_extra_questions:[]
that means no extra questions at all